pkg/sfu/buffer/rtpstats.go

```diff
@@ -799,17 +815,8 @@ func (r *RTPStats) SnapshotRtcpReceptionReport(ssrc uint32, proxyFracLost uint8,
 		return nil
 	}
 
-	packetsLost := uint32(0)
-	if r.params.IsReceiverReportDriven {
-		// receiver report driven should not be set for streams that need to generate reception report, but including code here for consistency
-		packetsLost = now.packetsLostOverridden - then.packetsLostOverridden
-		if int32(packetsLost) < 0 {
-			packetsLost = 0
-		}
-	} else {
-		intervalStats := r.getIntervalStats(uint16(then.extStartSN), uint16(now.extStartSN))
-		packetsLost = intervalStats.packetsLost
-	}
+	intervalStats := r.getIntervalStats(uint16(then.extStartSN), uint16(now.extStartSN))
+	packetsLost := intervalStats.packetsLost
```

Comment:
Cleaning these up. If receiver report based delta is needed, `DeltaInfoOverridden` should be used. So, cleaning these up.